extension/test/event-page-data-test.js

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) 2017 Google Inc. All rights reserved.
+// This code may only be used under the BSD style license found at
+// http://polymer.github.io/LICENSE.txt
+// Code distributed by Google as part of this project is also
+// subject to an additional IP rights grant found at
+// http://polymer.github.io/PATENTS.txt
+
+var assert = chai.assert;
+
+afterEach(function() {
+  target.innerHTML = '';
+});
+
+describe('EventPage', () => {
```

Comment:
Here and below, `mocha` best practices apparently are not to use the `() =>` thing for mocha functions like `describe` and `it` because it breaks use of `context` within:

https://mochajs.org/#arrow-functions

Reply:
Oh good to know! I've switched over.